SConstruct

```diff
@@ -435,6 +435,7 @@ def add_cluster_analysis(w):
                     ' --paths-relative-to ' + dataset_outdir(c) +
                     ' --namespace cft.cluster' +
                     ' --inferred-naive-name ' + options['inferred_naive_name'] +
+                   (' --ignore-timepoint-info ' if options['is_simulated_data'] else '') +
```

Comment:
Why are we doing this?

Reply:
Not sure exactly what you mean by `this`. If you mean why would we ever want to ignore timepoint info, according to @psathyrella simulated data does not have timepoints associated with samples, often times. So we are using this flag to tell process_partis.py to be aware of that.

Reply:
This was partly me being lazy and writing questions down as I have them; Should have read through the whole diff and then commented, as it would have been clearer what was going on. In any case, as stated below, I think this is the wrong tack.